events_test/src/test/java/com/adobe/aio/event/management/RegistrationServiceIntegrationTest.java

```diff
@@ -51,7 +51,7 @@ public void createGetDeleteJournalRegistration() {
     try {
       providerId = providerServiceTester.createOrUpdateProvider(TEST_EVENT_PROVIDER_LABEL,
           TEST_EVENT_CODE).getId();
-      Registration registration = createJournalRegistration(TEST_REGISTRATION_NAME, providerId,
+      Registration registration = createOrUpdateJournalRegistration(TEST_REGISTRATION_NAME, providerId,
```

Comment:
may be we could test both path ?
using `createOrUpdate` once and then `createOrUpdate` again ... once created 
here and may be in `ProviderServiceIntegrationTest` too 
wdyt ?

Reply:
I didn't get this

Reply:
done